InstallCob4.sh

```diff
@@ -35,7 +35,7 @@ function BasicInstallation {
   sudo apt-get install openssh-server -y --force-yes
   echo -e "\n${green}INFO:Let the server send a alive interval to clients to not get a broken pipe${NC}\n"
   echo "ClientAliveInterval 60" | sudo tee -a /etc/ssh/sshd_config
-  sudo sed -i 's/PermitRootLogin/PermitRootLogin yes/g' /etc/ssh/sshd_config
+  sudo sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/g' /etc/ssh/sshd_config
```

Comment:
is this from a file name? or a string?
should the `SPACE` be made explicit (ASCII)?